choose/command.go

```diff
@@ -98,7 +99,7 @@ func (o Options) Run() error {
 		}
 	}
 
-	fmt.Println(strings.TrimSuffix(s.String(), "\n"))
+	fmt.Print(s.String())
```

Comment:
I want to point out that, whether this PR will be accepted or not, this is still very reasonable (unless I'm missing something). Trimming the last "\n" and than adding it back again by using `fmt.Println` instead of just using `fmt.Print` does nothing but calls one additional function. 

In the case of this specific PR anyways, I added this because otherwise, when the `--optional` flag was passed, an extra new line was printed when nothing was selected, resulting in a wrong return ("\n" instead of "").

Reply:
That's a very good point! Thank you, we will definitely change this.

Reply:
This change is now on `main`: https://github.com/charmbracelet/gum/commit/81602545a65795d094fe7873d3548c22576f75b6

Reply:
Awesome! Thanks for the feedback, I'm glad this was added.